src/ResourceManagement/ResourceManager/Authentication/AzureCliCredentials.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Management.ResourceManager.Fluent.Authentication
+{
+    using Newtonsoft.Json;
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+
+    public class AzureCliCredentials
+    {
+        private static readonly string userProfile = "%USERPROFILE%";
```

Comment:
Are you sure it works both for windows and linux (and maybe mac)?

We got dotnet core that is cross platform.